modules/docs/arrow-docs/docs/docs/datatypes/reader/README.md

```diff
@@ -9,4 +9,214 @@ permalink: /docs/datatypes/reader/
 {:.intermediate}
 intermediate
 
-TODO. Meanwhile you can find a short description in the [intro to datatypes]({{ '/docs/datatypes/intro/' | relative_url }}).
+Reader is a datatype that represents computation that can read values from shared environment.
```

Comment:
can *inject* values from -> so people associate it with DI

from +a+ shared environment.